### PR TITLE
Second Contact Email page:

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
* Add the fields to the steps file & fields file
* Add the acceptance tests
* Add the translations
https://jira.digital.homeoffice.gov.uk/browse/FLHO-348

### CODE REVIEW

File: apps/shooting-clubs/acceptance/features/second-contact-email.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Scenario('check that an error appears if I submit empty fields', (
+  I,
+  secondContactEmailPage
+) => {
+  I.submitForm();
+  I.seeErrors(_.values(secondContactEmailPage.fields));
+});
+
+Scenario('I can go to the next page, second contact name page', (
```
Comment (bendiggle) [inline apps/shooting-clubs/acceptance/features/second-contact-email.js]: Not to second contact name page

Comment (sulthan-ahmed) [inline apps/shooting-clubs/acceptance/features/second-contact-email.js]: ahh good spot, it's because of copy paste

**Comment by bendiggle** [comment]
@sulthan-ahmed Needs rebasing

File: apps/shooting-clubs/acceptance/pages/second-contact-email.js [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+  url: 'second-contact-email',
+
+  fields: {
+    'second-contact-email-id': '#second-contact-email',
+    'second-contact-phone-id': '#second-contact-phone'
+  },
+
+  content: {
+    'valid-email': 'pokemon@gogo.com',
+    'invalid-email': 'pikachu!',
```
Comment (joefitter) [inline apps/shooting-clubs/acceptance/pages/second-contact-email.js]: lol

**Pull request merged**